### PR TITLE
fix(popover): avoid flash of unstyled content

### LINKED ISSUE #4427: Popover in 0.13.4 shows flash of unstyled content
Regression in 0.13.4 from 0.13.3 generates a FOUC on the popover directive.  It's very apparent on any page that has a slow digest cycle.  Making sure to hide element before the `$timeout` then promptly show works around the issue.

See plunkr to reproduce problem: http://plnkr.co/edit/XKtxNa67aaT2TsS7YcNV?p=preview

angular version: 1.4.5
angular-ui bootstrap version: 0.13.4
browser: latest stable Chrome

Workaround at https://github.com/bikegriffith/bootstrap/commit/af9d18df666a66a7c40c5409f89e7bdab073ef19 (I can pull request it if desired)

### PR DESCRIPTION
**Pull request by bikegriffith**
Fixes #4427

### CODE REVIEW

**Comment by RobJacobs** [comment]
This has been fixed under PR #4363 and PR #4379.  This updated [plunk](http://plnkr.co/edit/JERhdfUrRrOrzhHtkAYh?p=preview) shows the behavior with the latest code from master.

File: src/tooltip/tooltip.js [hunk 1]
```diff
@@ -149,8 +149,10 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
               if (!tooltip || !tooltip.html()) { return; }
 
               if (!positionTimeout) {
+                tooltip.hide(); // Hide to avoid FOUC until $timeout completes
```
Comment (bikegriffith) [inline src/tooltip/tooltip.js]: `positionTooltip` is called frequently in some cases for an already open tooltip, so this could cause flickering.

**Comment by bikegriffith** [comment]
Thanks @RobJacobs. Just realizing this while auditing the code again and testing against a hacked version of 0.13.4 in my app.  Will shut this down.

**Comment by RobJacobs** [comment]
@bikegriffith We're doing some more work to tighten up when the positioning method gets invoked and how often.  Next significant change is PR #4418.

**Pull request closed without merging**